proto/v1/org_policy_service.proto

```diff
@@ -297,12 +297,15 @@ message SensitiveData {
   string schema = 1;
   string table = 2;
   string column = 3;
-  SensitiveDataMaskType mask_type = 4;
-}
-
-enum SensitiveDataMaskType {
-  MASK_TYPE_UNSPECIFIED = 0;
-  DEFAULT = 1;
+  string data_category_id = 4;
+  enum MaskingLevel {
+    MASKING_LEVEL_UNSPECIFIED = 0;
+    DEFAULT = 1;
```

Comment:
I think DEFAULT = MASKING_LEVEL_UNSPECIFIED?

Reply:
Or is it MASKING_LEVEL_UNSPECIFIED = DEFAULT?

Reply:
Reuse the pre-defined masking level in common? https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/proto/v1/common.proto?L33